src/types/accessors/AbstractMetabolicModel.jl

```diff
@@ -369,3 +369,33 @@ By default, it should be safe to do nothing.
 function precache!(a::AbstractMetabolicModel)::Nothing
     nothing
 end
+
+"""
+$(TYPEDSIGNATURES)
+
+Return the molar mass of translated gene with ID `gid`.
+"""
+gene_protein_molar_mass(model::AbstractMetabolicModel, gid::String)::Maybe{Float64} = nothing
```

Comment:
this is normally called a "gene product", might be useful to stay with that.

Reply:
fixed